### PR TITLE
(doc) Fix typos in documentation

### PR DESCRIPTION
**Pull request by gaurav9822**

### CODE REVIEW

File: mypatch.patch [hunk 1]
```diff
@@ -0,0 +1,1081 @@
+diff --git a/log4j-core/src/main/java/org/apache/logging/log4j/core/Layout.java b/log4j-core/src/main/java/org/apache/logging/log4j/core/Layout.java
```
Comment (jvz) [inline mypatch.patch]: Could you remove this file?

File: src/site/asciidoc/index.adoc.rej [hunk 2]
```diff
@@ -0,0 +1,21 @@
+diff a/src/site/asciidoc/index.adoc b/src/site/asciidoc/index.adoc	(rejected hunks)
```
Comment (jvz) [inline src/site/asciidoc/index.adoc.rej]: Could you resolve whatever merge issue this is?

**Comment by rgoers** [comment]
Wow. It must have taken some time to find and correct all of these. Thanks. However, this fixes the master branch, which is still a work in progress. I suspect many of these errors also exist in the release-2.x branch.

**Comment by gaurav9822** [comment]
@rgoers Thank you for the kind words. Would love to contribute more to log4j2.

**Comment by pzygielo** [comment]
> Would love to contribute more to log4j2.

Could you provide PR to remove those new files @jvz had questions about?

**Comment by rgoers** [comment]
@pzygielo No, because I have no idea what Matt was concerned about. There is no reason he would want Layout.java to be removed.

**Comment by pzygielo** [comment]
> @pzygielo No, because I have no idea what Matt was concerned about. There is no reason he would want Layout.java to be removed.

*  mypatch.patch
*  src/site/asciidoc/index.adoc.rej

**Comment by rgoers** [comment]
Thanks, I have removed them.

**Comment by gaurav9822** [comment]
Thank you Ralph.

**Pull request merged**